src/Adapters/Storage/Mongo/MongoSchemaCollection.js

```diff
@@ -1,5 +1,6 @@
 
 import MongoCollection from './MongoCollection';
+import { defaultColumns } from '../../../Schema';
```

Comment:
not really happy with that circular dependency, nor the fact that the adapter depends upon parse-server... but otherwise that would defer the responsibility to the router (which is not good). Solution would be to introduce a SchemaController